src/gatsby-node.ts

```diff
@@ -128,6 +155,11 @@ export const onPostBuild = ({ store }: any, userPluginOptions: PluginOptions) =>
         JSON.stringify(slsRoutingRules)
     );
 
+    fs.writeFileSync(
+        path.join(program.directory, './.cache/s3.permanentRedirects.json'),
```

Comment:
Similarly, I think the name of this file could be changed. Maybe `s3.objectRedirects.json`?